CTFd/utils/migrations/__init__.py

```diff
@@ -20,22 +19,24 @@ def create_database():
     if url.drivername.startswith("mysql"):
         url.query["charset"] = "utf8mb4"
 
-    # Creates database if the database database does not exist
-    if not database_exists_util(url):
+    # Creates database if the database does not exist
+    try:
+        # If the user don't have permission to access to "postres" databasen  this
+        # will fail: https://github.com/kvesteri/sqlalchemy-utils/pull/372
+        # in that case we asume that the database was previously created and we can't
+        # drop or create the database
+        exists = database_exists_util(url)
```

Comment:
Even if we ignore the Postgres support discussion, I don't know if I would accept this change. Too hacky for my tastes and probably could affect other things. Maybe better to only exception handle under Postgres. 

We would need to wait for https://github.com/kvesteri/sqlalchemy-utils/pull/372